ocm.py

```diff
@@ -0,0 +1,44 @@
+#!/bin/python
+import requests
+
+def call_ocm(endpoint, token, parameters=None):
+    requests.packages.urllib3.disable_warnings()
```

Comment:
this should happen at the module scope, before this function definition.  We don't need to call it in every `call_ocm`, do we?